### DISCUSSION TITLE
Newfoundland and Labrador Data?

### DISCUSSION THREAD

**Discussion by pjakobsen** [electricitymaps/electricitymaps-contrib#3306] [Q&A]
I'm keeping an eye on the Muskrat Falls project, another [5.2 billion dollars was injected](https://www.theglobeandmail.com/business/article-ottawa-newfoundland-agree-to-52-billion-deal-on-muskrat-falls/) a couple of weeks ago. 

Does anyone know if open electricity data is forthcoming from Newfoundland and Labrador?   

Who is best equipped to know which provinces and territories may have new data sources coming in the near future.   Is there any use is contacting each jurisdiction, or has someone already done this?

**Comment by pbubik** [comment]
In Canada, each province and territory is responsible for their own grids. I have been in touch with Manitoba, Saskatchewan and BC to bug them to make their data public. It's like pushing a rope, but I think if enough of us will bug them, eventually they will cave in. Quebec did. :)
Manitoba gave us enough to correct their export average, BC is looking into it, but needs to wrap their heads around what to do with individual utilities (Fortis area in SE BC and Fort Nelson area in the north - the data doesn't belong to BC Hydro, so they will need to grapple with it for a while). I have been e-mailing with alf.leake@bchydro.com - e-mail him that you would like to see this data too - perhaps it will give him the push he needs.

I think contacting each juridstiction and bugging them is the only way we will get more data public.

**Reply by pjakobsen** [reply]
Hmm, with a Federal election coming up, almost every candidate will brag that they are green. The lack of open electricity data  can could be a nice comeback zinger during a debate that will no doubt include lots of NetZero waffling.

**Comment by jarek** [comment]
> I'm keeping an eye on the Muskrat Falls project, another 5.2 billion dollars was injected a couple of weeks ago.
> Does anyone know if open electricity data is forthcoming from Newfoundland and Labrador?

Unfortunately, boondoggles like this are usually a reason why open data is _not_ published. Real data makes damage-control spin harder to pull off.

Usually the best bet would be to wait for a change of government and apply the usual arguments to the new people in power right afterwards (transparency, showing off how clean it is - whatever might work). But in this case I think both major parties were involved?
